internal/policies/mount/mount_test.go

```diff
@@ -115,15 +116,17 @@ func TestApplyPolicy(t *testing.T) {
 
 		/**************************** SYSTEM ***************************/
 		// Success cases.
-		"system, successfully apply policy for entry with one value":       {isComputer: true},
-		"system, successfully apply policy for entry with multiple values": {entries: []string{"entry with multiple values"}, isComputer: true},
-		"system, successfully apply policy for entry with repeated values": {entries: []string{"entry with repeated values"}, isComputer: true},
-		"system, successfully apply policy filtering out unsupported keys": {entries: []string{"entry with multiple values", "entry with one value"}, keys: []string{"unsupported", "system-mounts"}, isComputer: true},
+		"system, successfully apply policy for entry with one value":              {isComputer: true},
+		"system, successfully apply policy for entry with multiple values":        {entries: []string{"entry with multiple values"}, isComputer: true},
+		"system, successfully apply policy for entry with repeated values":        {entries: []string{"entry with repeated values"}, isComputer: true},
+		"system, successfully apply policy for entry with repeated tagged values": {entries: []string{"entry with repeated tagged values"}, isComputer: true},
+		"system, successfully apply policy filtering out unsupported keys":        {entries: []string{"entry with multiple values", "entry with one value"}, keys: []string{"unsupported", "system-mounts"}, isComputer: true},
 
 		// Special cases.
 		"system, successfully apply policy with kerberos tagged values":                         {entries: []string{"entry with kerberos auth tags"}, isComputer: true},
 		"system, successfully apply policy prioritizing the first value found, despite the tag": {entries: []string{"entry with same values tagged and untagged"}, isComputer: true},
 		"system, only emit a warning when starting new units and systemctl fails":               {isComputer: true, firstSystemCtlFailingArgs: []string{"start"}},
+		"system, only emit a warning when disabling previous units and systemctl fails":         {isComputer: true, secondCall: []string{"entry with multiple values"}, secondSystemCtlFailingArgs: []string{"stop"}},
```

Comment:
I think the message is misleading, you want to tell that it’s stopping which fails

Reply:
oops